src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. `new Random()` and `new ColorSupplier()` object will be created on each `getRandomBall()` call. How can you avoid this?